plugins/aws/elasticache/elasticacheRedisMultiAZ.js

```diff
@@ -0,0 +1,63 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'ElastiCache Redis Cluster Have Multi-AZ',
+    category: 'ElastiCache',
+    domain: 'Databases',
+    description: 'Ensure that your ElastiCache Redis Cache clusters are using a Multi-AZ deployment configuration to enhance High Availability.',
+    more_info: 'Enabling the Multi-AZ feature for your Redis Cache clusters will improve the fault tolerance in case the read/write primary node becomes unreachable due to loss of network connectivity, loss of availability in the primary’s AZ, etc. ',
```

Comment:
```suggestion
    more_info: 'Enabling the Multi-AZ feature for your Redis Cache clusters will improve the fault tolerance in case the read/write primary node becomes unreachable due to loss of network connectivity, loss of availability in the primary\'s AZ, etc. ',
```